services/ui-src/src/components/tables/EntityRow.tsx

```diff
@@ -29,12 +29,13 @@ export const EntityRow = ({
   openAddEditEntityModal,
   openDeleteEntityModal,
   openOverlayOrDrawer,
+  stepType,
```

Comment:
just a slight recommendation to cut down on adding new props, you can "formEntity.stepType" to get step type information